src/main/java/org/broadinstitute/hellbender/tools/walkers/coverage/DepthOfCoverage.java

```diff
@@ -62,6 +62,7 @@
  * <ul>
  *     <li>DepthOfCoverage currently only supports typical nucleotide (and N) bases, IUPAC ambiguity codes or other non-ATCGN bases will cause exceptions</li>
  *     <li>Read filters are applied to the reads before being counted in coverage information. By default Duplicate Marked and non-primary alignments are not counted. This can be disabled with --disable-tool-default-read-filters.</li>
+ *     <li>In order to filter reads out by their mapping qualities, the recommended approach is to use the MappingQualityReadFilter with the --minimum-mapping-quality argument specified</li>
```

Comment:
We might want to suggest using the MappingQualityAvailableReadFilter, since mapping quality 255 (mapping quality unavailable) always satisfies the minimum mapping quality threshold.

Reply:
hmm... interesting, it seems that mapping-quality unavailable was not being handled at all before (I would have expected it to be part of is mapped or WellFormedReadFilter). I'm hesitant to add too much to this comment, I included this here because there was a commonly used "-mmq" argument in GATK3 that wasn't ported. I will add a note that max-mapping-quality exists as well, GATK3 defaulted to MAXINT as its maximum so it wasn't handled by default.